### PR TITLE
HOTFIX: Setting default values to true

### PR DESCRIPTION
**Pull request by claesgill**
## Closes/fixes/resolves issue(s)?
None

## What was added/changed/fixed?
`use_canary` and `use_vault_provider` is default `true`

## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [x] Added to project
- [ ] ~~Added to milestone~~

### CODE REVIEW

File: .github/workflows/on_pr_push_master.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Super-Linter
-        uses: github/super-linter@latest
+        uses: github/super-linter@v3.13.1
```
Comment (zhenik) [inline .github/workflows/on_pr_push_master.yml]: Not sure that it will help due to this issue https://github.com/github/super-linter/issues/708

Comment (claesgill) [reply .github/workflows/on_pr_push_master.yml]: Tried to fix accordingly to a comment about v3.13.1: https://github.com/github/super-linter/issues/938 

But it didn't seem to fix it anyway :shrug:

Comment (claesgill) [reply .github/workflows/on_pr_push_master.yml]: Here's the comment btw: https://github.com/github/super-linter/issues/938#issuecomment-720769293

Comment (zhenik) [reply .github/workflows/on_pr_push_master.yml]: For now, you can use @Neha-Sinha2305 fork https://github.com/Neha-Sinha2305/super-linter

**Review by zhenik — DISMISSED** [review]

**Review by pdmthorsrud — DISMISSED** [review]

**Review by dangernil — APPROVED** [review]

**Review by zhenik — APPROVED** [review]

**Pull request merged**